### ISSUE TITLE
MineMeld End-of-Life (in Autofocus)

### ISSUE THREAD

**Issue by idev** [PaloAltoNetworks/minemeld#60]
Hello MineMeld Users,

I am wondering if anybody else is concerned by the move of Palo Alto no longer providing a MineMeld instance via Autofocus.
Are there any alternatives to self-hosting a MineMeld Server, like MineMeld as a Service (Saas)?

Kind regards

**Comment by xhoms** [comment]
What about running its [containerized](https://hub.docker.com/r/paloaltonetworks/minemeld) version in any managed container infrastructure like AWS's ECS?

**Comment by idev** [comment]
The idea of SaaS is, not to be responsible for operations in any form. A containerized version would delegate this responsibility back to the "owner".

**Comment by xhoms** [comment]
Fair enough. AFAIK Palo Alto Networks' [XSOAR TIM](https://www.paloaltonetworks.com/cortex/threat-intel-management) is the supported product for Thread Intel Management and it can be delivered as a [fully hosted solution](https://www.paloaltonetworks.com/resources/datasheets/cortex-xsoar-hosted-solution-datasheet).

I'm not aware of any company providing open source MineMeld as a managed service.

**Comment by idev** [comment]
Thank you! I am aware of XSOAR TIM, but as far as I know this only provides a subset of the features Minemeld has?!
Main feature is generating a IP / URL / Domain List from sources to feed a dynamic firewall rule (EDL).

**Comment by xhoms** [comment]
I'm afraid XSOAR TIM might be a super-set instead. Maybe too featured for a basic EDL feed application.
